src/pages/Organization/AdminLogin.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { FaGoogle, FaRegEnvelope, FaRegEye } from 'react-icons/fa';
+import { MdLockOutline } from 'react-icons/md';
+
+const AdminLogin = () => {
+  return (
```

Comment:
const [visible,setVisible]=useState(false)

//state manager to keep track of visitor choice